index.js

```diff
@@ -15,13 +12,8 @@ const isStaticExportProject = require('./helpers/isStaticExportProject')
 // - Between the build and postbuild steps, any functions are bundled
 
 module.exports = {
-  async onPreBuild({ netlifyConfig, packageJson: { scripts = {}, dependencies = {} }, utils }) {
+  async onPreBuild({ netlifyConfig = {}, packageJson: { scripts = {}, dependencies = {} }, utils }) {
     const { failBuild } = utils.build
-
-    if (!(await hasFramework('next'))) {
```

Comment:
Until this is made a core plugin, if a user is installing this plugin but they forgot to install `next`, wouldn't it be better to print this error message?